pkg/mutation/system.go

```diff
@@ -107,12 +108,20 @@ func (s *System) Mutate(obj *unstructured.Unstructured, ns *corev1.Namespace) (b
 			}
 			if cmp.Equal(original, obj) {
 				if *MutationLoggingEnabled {
-					logAppliedMutations("Oscillating mutation.", original, allAppliedMutations)
+					logAppliedMutations("Oscillating mutation.", original, allAppliedMutations, nil)
 				}
 				return false, fmt.Errorf("oscillating mutation for %s %s %s %s", obj.GroupVersionKind().Group, obj.GroupVersionKind().Kind, obj.GetNamespace(), obj.GetName())
 			}
+			mutationTimestamp := time.Now()
 			if *MutationLoggingEnabled {
-				logAppliedMutations("Mutation applied", original, allAppliedMutations)
+				logAppliedMutations("Mutation applied", original, allAppliedMutations, &mutationTimestamp)
```

Comment:
we should use a unique mutation ID rather than a timestamp... multiple golang threads, multiple pods, there is no guarantee these wont be duplicated.

Maybe UUID?

Reply:
Added a mutation uuid